### PR TITLE
Upgrade Sentry package; handle most errors

### PR DESCRIPTION
**Pull request by RoyEJohnson**

### CODE REVIEW

File: src/app/components/shell/header/menus/upper-menu/upper-menu.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ function BlogItem() {
         'pages?type=news.newsArticle&fields=id&limit=3'
     );
 
-    const shouldDisplay = data && data.items.length > 0;
+    const shouldDisplay = data && data.items && data.items.length > 0;
```
Comment (nathanstitt) [inline src/app/components/shell/header/menus/upper-menu/upper-menu.js:13]: BTW, tutor's using optional chaining which makes this: 
`const shouldDisplay = !!data?.items?.length`
https://github.com/openstax/tutor-js/blob/main/babel.config.js#L16

File: src/app/components/shell/shell.js [hunk 2]
```diff
@@ -44,7 +44,10 @@ function ImportedComponent({name}) {
 
     React.useEffect(
         () => import(`./${name}/${name}`)
-            .then((content) => setContent(<content.default />)),
+            .then(
+                (content) => setContent(<content.default />),
+                (err) => console.warn(`Error loading page ${name}:`, err)
```
Comment (nathanstitt) [inline src/app/components/shell/shell.js:49]: same, should it log to sentry?

File: src/app/components/shell/router.js [hunk 3]
```diff
@@ -134,10 +134,15 @@ function ImportedPage({name}) {
...
-            setContent(<content.default />);
-            window.scrollTo(0, 0);
-        });
+        import(`~/pages/${name}/${name}`).then(
+            (content) => {
+                setContent(<content.default />);
+                window.scrollTo(0, 0);
+            },
+            (err) => {
+                console.warn(`Unable to import ${name}`, err);
```
Comment (nathanstitt) [inline src/app/components/shell/router.js:143]: should this log to sentry?

**Review by nathanstitt — APPROVED** [review]

File: src/app/components/shell/header/menus/upper-menu/upper-menu.js (hunk 1 above)
Comment (RoyEJohnson) [reply src/app/components/shell/header/menus/upper-menu/upper-menu.js:13]: Woah: This plugin is included in @babel/preset-env

**Pull request merged**